apprentice_teacher.rb

```diff
@@ -26,15 +17,6 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
   def set_performance_rating(rating)
     response = ""
     if rating > 80
```

Comment:
set_performance_rating is still common between the ApprenticeTeacher and SeniorTeacher classes. How will you extract it ?